src/utils.js

```diff
@@ -135,9 +135,18 @@ export function standardizeWitUnits(
   }
   const num = Big(amount)
   const exponent = witUnitConversor[inputCurrency][outputCurrency]
-  return outputCurrency === WIT_UNIT.NANO
-    ? num.times(Big(10).pow(exponent), 0).toFixed(0)
-    : num.times(Big(10).pow(exponent), 0).toFixed(truncate)
+  const result = num.times(Big(10).pow(exponent), 0)
+
+  // console.log(result.toString(), result.cmp(1))
+  if (truncate === -1) Number(result.toString)
```

Comment:
```suggestion
  if (truncate === -1) {
    return Number(result.toString)
  }
```

And personally I would use `null` as the special value instead of `-1`